test/Dialect/FIRRTL/xmr.mlir

```diff
@@ -0,0 +1,109 @@
+// RUN: circt-opt %s -split-input-file
```

Comment:
Since we're moving away from "XMR" generally, should this be "ref.mlir" or something instead of "xmr.mlir"?
That said, looks like we're not losing the "XMR" association entirely so :shrug:.

Reply:
Lets do it `ref.mlir`